games/game_darkestdungeon.py

```diff
@@ -26,12 +26,5 @@ def executables(self):
         if not path.exists():
             path = QFileInfo(self.gameDirectory(), "_windowsnosteam/darkest.exe")
         return [
-            mobase.ExecutableInfo("Darkest Dungeon", path),
+            mobase.ExecutableInfo("Darkest Dungeon", path).withWorkingDirectory(self.gameDirectory()),
         ]
-
-    def savesDirectory(self):
```

Comment:
Any reason why you removed the saves directory?

Reply:
There are different possible save locations:
- Steam + Steam Cloud Enabled: `Steam/userdata/<your_steam_id>/262060/remote`
- GOG or Steam + Steam Cloud Disabled: `%DOCUMENTS%/Darkest`

Then there is the problem with the save files themselves. In the save folder you have 10 sub-directories (`profile_i` with `0 <= i <= 9`) that represent 1 "save slot". Instead of having 1000 save files like in Skyrim you have 1 save slot that gets overwritten each time. Each of those profile sub-folders also contains 7 different JSON files with the actual save data.

Since MO2 was made with the save file system from BGS in mind, it doesn't work well for a save slot system.

Reply:
MO2 can handle save slot. The only thing MO2 could not handle would be multiple saves in one file, but that's not the case here I guess?

For the various save locations, I guess you check. I'll see if I can add utility to retrieve the version (GOG / Steam) and the Steam Cloud Enabled or not.

Reply:
Can MO2 also handle save slot folders or save folders in general? Save slots look like this:
```text
C:\Program Files (x86)\Steam\userdata\149956546\262060\remote
│   steam_init.json
├───profile_0
│       persist.estate.json
│       persist.game.json
│       persist.loading_screen.json
│       persist.map.json
│       persist.raid.json
│       persist.roster.json
│       persist.upgrades.json
│       
└───profile_9
        novelty_tracker_mp.json
        persist.banner_custom.json
        persist.circus_estate.json
        persist.game.json
        persist.mp_progression.json
        persist.narration.json
        persist.prize_booth.json
        persist.rankings.json
        persist.roster.network.json
        persist.town_event.json
        persist.tutorial.json
        
```

Reply:
If each save slot (`profile_0`, `profile_1`, etc.) corresponds to a single save, then yes. I don't know DD, but if it's some kind of automatic-save by profile, then it should be fine. What MO2 cannot handle would be something like this:

```
profile1/
    all_saves_in_one_file.xyz
profile2/
    all_saves_in_one_file.xyz
```